lib/go-tc/origins.go

```diff
@@ -54,3 +56,74 @@ type Origin struct {
 	Tenant            *string    `json:"tenant" db:"tenant"`
 	TenantID          *int       `json:"tenantId" db:"tenant_id"`
 }
+
+// OriginsResponseV5 is an alias for the latest minor version of the major version 5.
+type OriginsResponseV5 = OriginsResponseV50
+
+// OriginsResponseV50 is a list of Origins as a response for APIv5.
+type OriginsResponseV50 struct {
+	Response []OriginV5 `json:"response"`
+	Alerts
+}
+
+// OriginDetailResponseV5 is an alias for the latest minor version of the major version 5.
+type OriginDetailResponseV5 = OriginDetailResponseV50
+
+// OriginDetailResponseV50 is the JSON object returned for a single origin in APIv5.
+type OriginDetailResponseV50 struct {
+	Response OriginV5 `json:"response"`
+	Alerts
+}
+
+// OriginV5 is an alias for the latest minor version of the major version 5.
+type OriginV5 = OriginV50
+
+// OriginV50 contains information relating to an Origin, in the latest minor version APIv50.
+type OriginV50 struct {
+	Cachegroup        *string    `json:"cachegroup" db:"cachegroup"`
+	CachegroupID      *int       `json:"cachegroupId" db:"cachegroup_id"`
+	Coordinate        *string    `json:"coordinate" db:"coordinate"`
+	CoordinateID      *int       `json:"coordinateId" db:"coordinate_id"`
+	DeliveryService   *string    `json:"deliveryService" db:"deliveryservice"`
+	DeliveryServiceID *int       `json:"deliveryServiceId" db:"deliveryservice_id"`
+	FQDN              *string    `json:"fqdn" db:"fqdn"`
+	ID                *int       `json:"id" db:"id"`
+	IP6Address        *string    `json:"ip6Address" db:"ip6_address"`
+	IPAddress         *string    `json:"ipAddress" db:"ip_address"`
+	IsPrimary         *bool      `json:"isPrimary" db:"is_primary"`
+	LastUpdated       *time.Time `json:"lastUpdated" db:"last_updated"`
+	Name              *string    `json:"name" db:"name"`
```

Comment:
These values are never allowed to be `nil`, so they shouldn't be reference types